modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -202,45 +169,44 @@ output application/json
   ]
 }
 ----
-
-
-//com.mulesoft.dw.stacksize//
-|`com.mulesoft.dw.stacksize`|
-Sets the maximum size of the stack. When a function recurses too deeply, DataWeave throws an error, such as `Stack Overflow. Max stack is 256.`
-
-* Type: `Number`
-* Default: `256`
-
-
-//com.mulesoft.dw.xml.supportDTD//
-|`com.mulesoft.dw.xml.supportDTD`|
-Controls whether DTD handling is enabled or disabled. When this property is set to `false`, DataWeaves skips processing of both internal and external subsets. Note that the default for this property changed from `true` to `false` in Mule version 4.3.0-20210427, which includes the May, 2021 patch of DataWeave version 2.3.0.
-
 * Type: `Boolean`
 * Default: `false`
-
-
-//com.mulesoft.dw.track.cursor.close//
-|`com.mulesoft.dw.track.cursor.close`|
-When set to `true`, tracks the stack trace from which the `CursorProvider#close()` method is called.
-Use this property for troubleshooting, for example, if `CursorProvider#openCursor()` is called on a cursor that is already closed.
-
+Available for language levels: 2.3, 2.4
+//com.mulesoft.dw.workingdirectory.base//
+|`com.mulesoft.dw.workingdirectory.base`|
+Specifies the base directory in which temporary files will be written
+* Type: `String`
+* Default: `/var/folders/8f/5jqs1kh97k16dzsmxxph2m5c0000gp/T/`
+//com.mulesoft.dw.workingdirectory.delete_sync//
+|`com.mulesoft.dw.workingdirectory.delete_sync`|
+If set, file deletion is done synchronously
 * Type: `Boolean`
 * Default: `false`
-
-
-//com.mulesoft.dw.verbose//
-|`com.mulesoft.dw.verbose`|
```

Comment:
com.mulesoft.dw.verbose removed? is this intended? there was no warning that it could be removed.

Reply:
asked on channel

Reply:
Removed Feb 2021 in 2.4

Reply:
TODO: update 2.4 docs about removal